static/app/components/arithmeticBuilder/token/function.tsx

```diff
@@ -90,26 +83,177 @@ export function ArithmeticTokenFunction({
   );
 }
 
-interface InternalInputProps extends ArithmeticTokenFunctionProps {
+type Argument = {attribute: string; key: string; text: string};
```

Comment:
If I'm reading this right, a better name for the attributes of this type might be
```suggestion
type Argument = {label: string; key: string; value string};
```